src/dbnode/storage/index/types.go

```diff
@@ -74,44 +74,48 @@ func (o QueryOptions) LimitExceeded(size int) bool {
 	return o.Limit > 0 && size >= o.Limit
 }
 
-// QueryResults is the collection of results for a query.
-type QueryResults struct {
+// QueryReturnResults is the collection of results for a query.
+type QueryReturnResults struct {
 	Results    Results
 	Exhaustive bool
 }
 
-// Results is a collection of results for a query, it is synchronized
+// Results is a collection of results for a generic query, it is synchronized
```

Comment:
Can we rename this to `GenericResults` or `BaseResults`, then rename `QueryResults` back to `Results` so we can avoid adding the word `"Return`" into `QueryReturnResults` (and the aggregate versions of that struct)?

Reply:
Sure, was having a bit of trouble getting good names in as you can see :p